gems/zipruby/CVE-2012-1163.yml

```diff
@@ -0,0 +1,15 @@
+---
+gem: zipruby
+cve: 2012-1163
+url: https://packetstormsecurity.com/files/cve/CVE-2012-1163
+title: |
+  libzip (version <= 0.10) has two vulnerabilities that may lead to a heap
+  overflow or an information leak via corrupted zip files
+date: 2012-03-21
+description: |
+  libzip (version <= 0.10) has two vulnerabilities that may lead to a heap
+  overflow or an information leak via corrupted zip files. PHP (versions
+  5.4.0 and <= 5.3.10) and the Ruby binding zipruby (version <= 0.3.6) are
+  also affected as they include copies of affected libzip versions.
+patched_versions:
+  - ">= 0.3.6"
```

Comment:
There are no versions currently released that are above this, should we really be putting this as patched versions? We can't guarantee that there will ever be a fixed version released...